tests/streamlit_ui_tests/graph.spec.ts

```diff
@@ -167,41 +173,56 @@ test.describe('Graph Functionality', () => {
     await expect(mito.locator('.legend')).toHaveText('variableColumn1 Column2');
   });
 
-  test('Make a histogram and change the aggregation function', async ({ page }) => {
+  test('Make a histogram and change the histogram specific configurations', async ({ page }) => {
     const mito = await getMitoFrameWithTestCSV(page);
     
-    await clickButtonAndAwaitResponse(page, mito, { name: 'Graph' })
-
-    await expect(mito.locator('.g-gtitle', { hasText: 'Column1 bar chart' })).toBeVisible();
-    await mito.getByRole('button', { name: '▾ Change Chart Type' }).click();
-    await mito.getByRole('button', { name: 'Histogram' }).hover();
-    await mito.getByRole('button', { name: 'Grouped' }).first().click();
+    await openGraphEditor(mito, page);
+    await changeChartType(mito, page, 'Histogram', 'Grouped', 0);
 
+    // Change the aggregation function
     await mito.locator('.mito-graph-configuration-container').getByText('count').click();
     await mito.locator('.mito-dropdown-item', { hasText: 'sum' }).click();
     await expect(mito.locator('.g-ytitle')).toHaveText('sum of None');
+
+    // Change the bin size
+    await expect(mito.locator('g.point')).toHaveCount(6);
+    await mito.locator('.mito-graph-configuration-container').locator('input[type="number"]').fill('1');
+    await awaitResponse(page);
+    await expect(mito.locator('g.point')).toHaveCount(4);
```

Comment:
I think some comments here to describe how we know that there should be a count of 6 and 4 would be useful